tools/autowiki/autowiki.js

```diff
@@ -0,0 +1,83 @@
+const fs = require("fs").promises
+const MWBot = require("mwbot")
+
+const { USERNAME, PASSWORD } = process.env
+
+if (!USERNAME) {
+	console.error("USERNAME was not set.")
+	process.exit(1)
+}
+
+if (!PASSWORD) {
+	console.error("PASSWORD was not set.")
+	process.exit(1)
+}
+
+const PAGE_EDIT_FILENAME = process.argv[2]
+
+if (!PAGE_EDIT_FILENAME) {
+	console.error("No filename specified to edit pages")
+	process.exit(1)
+}
+
+const FILE_EDIT_FILENAME = process.argv[3]
+
+if (!FILE_EDIT_FILENAME) {
+	console.error("No filename specified to edit files")
+	process.exit(1)
+}
+
+async function main() {
+	console.log(`Reading from ${PAGE_EDIT_FILENAME}`)
+	const editFile = await (await fs.readFile(PAGE_EDIT_FILENAME, "utf8")).split("\n")
+
+	console.log(`Logging in as ${USERNAME}`)
+
+	const bot = new MWBot()
+
+	await bot.loginGetEditToken({
+		apiUrl: "https://tgstation13.org/wiki/api.php",
```

Comment:
goes to tgstation wiki lol

Reply:
might need something on wiki side to correctly link and generate stuff